fix scrollbar for RTL languages

Thatkookooguy opened the pull request:
Fix #1149

In the current style configuration (for desktop), RTL languages are overlapping with the emoji icon.

Here's an example:
LTR before:
![ltr_before](https://cloud.githubusercontent.com/assets/10427304/15448840/a6e9ac78-1f75-11e6-819d-f4f1c12b407c.PNG)

RTL before:
![before](https://cloud.githubusercontent.com/assets/10427304/15448843/ace3d428-1f75-11e6-9623-0156242b7d39.PNG)

LTR after:
![ltr_after](https://cloud.githubusercontent.com/assets/10427304/15448844/b505eeac-1f75-11e6-8e5c-9c269235c944.PNG)

RTL after:
![after](https://cloud.githubusercontent.com/assets/10427304/15448845/b91ec266-1f75-11e6-94de-44ab30fbbbf3.PNG)

though it looks liks the scrollbar is useless since the container expands when you write.

should it be removed alltogether? or after a certain amount of lines it becomes active?

thx for the great tool! :-)

Thatkookooguy commented:
well... checked out the original behavior and copied it to work for both RTL and LTR languages

Thatkookooguy commented:
hello?

CarlitoBriganteNT commented:
на тв телеграмм как сделаь

Thatkookooguy commented:
I'm not sure what you wrote.

when I put it inside google translate I get: на тв телеграмм как сделаь -> on how to make a tv telegrams

mokeev1995 commented:
@Thatkookooguy I'm native speaker and I didn't understang what he wrote too. It's not your fault. (Or it's offtop and it can be translated like: "How to make it work on my TV?")

zhukov commented:
Wow. I didn't expect it, but it seems to work.

1. This 19px additional padding to left and right, I guess it's a scrollbar width. It should be calculated dynamically in Javascript, like was in app/js/message_composer.js before your changes. You just apply the style to the same element.
2. Emoji tooltip is not showing up on hover. Can be fixed by adding z-index to the button, but I'm not sure it's a best solution.
3. Please squash and rebase after fixing https://github.com/zhukov/webogram/blob/master/CONTRIBUTING.md#how-to-implement-changes-suggested-on-a-pull-request

zhukov commented:
@Thatkookooguy ping

Thatkookooguy commented:
cool. I'll work on it this week and update the PR

zhukov commented:
@Thatkookooguy ping again :)

Thatkookooguy commented:
@zhukov sorry 😄  forgot about this! fixing this soon 😇

Thatkookooguy commented:
@zhukov this `git fetch upstream` doesn't seem to work for me. I'm getting the following error:
```
fatal: 'upstream' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

I have the original repository on as `forkedFrom` and my repository as `origin`

any suggestions?

zhukov commented:
@nukeador since you are the author of original CONTRIBUTING.MD file, could you help us a bit? 🤓

nukeador commented:
Make sure you have followed all the instructions and you have done:

``git remote add upstream git@github.com:zhukov/webogram.git``

In order to have the original repo named as upstream. Or use the forkedFrom instead of upstream in your case if this points to the same url.

Thatkookooguy commented:
@nukeador thanks! it worked 👍

Ryuno-Ki commented:
@Thatkookooguy Merge conflicts.

Thatkookooguy commented:
yes. haven't finished this yet

Thatkookooguy commented:
@Ryuno-Ki fixed the merge conflicts with a rebase as suggested in `CONTRIBUTING.MD`

@zhukov I changed the solution a bit to follow your instructions. Now, the same mechanism calculates the right properties to keep both RTL and LTR scrollbars hidden.

Also added a little comment to explain a bit the calculations.

Thatkookooguy commented:
sorry it took so long 😅

zhukov commented:
Thanks for the PR.
